.github/workflows/composite/notify-slack/action.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Notify Slack
+description: Send a notification to slack about build status
+
+inputs:
+  webhook:
+    description: Slack incomming webhook specifying the channel to send to.
+  status:
+    description: Success status with valid input 'success' and 'failure'.
+  message:
+    description: Message to be displayed in notification.
+
+runs:
+  using: composite
+  # env is not allowed in composite actions
+  steps:
+    - name: Set layout for default
```

Comment:
Do you have a screenshot or link to the output in Slack for a demo run for each scenario?

Reply:
For the success scenario there is an an example at https://magmacore.slack.com/archives/C01RGBRKGRJ/p1668177629724539. I did not test the others in particular, because they are than expected to work similarly.